spec/rails_helper.rb

```diff
@@ -29,8 +29,12 @@ def create_category
     let!(:category) { Category.create(name: 'Lenses') }
   end
 
+  def create_brand
+    let!(:brand) { Brand.create(name: 'Canon') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.